api.go

```diff
@@ -0,0 +1,118 @@
+package main
```

Comment:
Most of my pairing bot changes involve querying the RC api. I created this file to hold all that API calling logic.